extension/ezjscore/classes/ezjscpacker.php

```diff
@@ -501,6 +501,9 @@ static function fixImgPaths( $fileContent, $file )
            // Pop the css file name
            array_pop( $cssPathArray );
            $cssPathCount = count( $cssPathArray );
+           // sort by url length
```

Comment:
CS. (sorry for so many nitpicks) The existing comments all start with an upper case letter and I think it would be best to follow this convention within your additions.